ansys/mapdl/core/math.py

```diff
@@ -452,6 +452,79 @@ def set_vec(self, vname, data):
     def rhs(self, type=np.double, fname="file.full"):
         return self.getVec(type, fname, "RHS")
 
+    def svd(self, M, thresh=None, sig=None, V=None):
+        """Apply an SVD Algorithm on a matrix
+
+        Parameters
+        ----------
+        M : ansys.AnsMat
+            The array to compress.
+
+        thresh:
+            
+        sig : str, optional
+        V   : str, optional
+            
+        Examples
+        --------
+        Apply SVD on an existing Dense Rectangular Matrix, using default threshold.
+        The M matrix is modified in-situ. 
+
+        >>> mm.svd( M)
+        """
+
+        if not thresh:
+            thresh = ''
+
+        if not sig:
+            sig = ''
+
+        if not V:
+            V = ''
+
+        self._mapdl.run("*COMP," + M.id + ",SVD," + str(thresh) + ',' + str(sig) + ',' + str(V))
```

Comment:
```suggestion
        self._mapdl.run(f"*COMP,{M.id},SVD,{thresh},{sig},{V}")
```
F-strings are a bit more readable.